install_gcloud.sh

```diff
@@ -13,6 +15,13 @@ if [[ ! -d "${HOME}/google-cloud-sdk/bin" ]]; then
 
 fi
 
-# Verify installation succeeded.
 source "${HOME}/google-cloud-sdk/path.bash.inc"
+
+# Install components
+gcloud config set core/disable_prompts true
+for COMPONENT in $*; do
```

Comment:
Please use `"$@"` instead of `$*`.

See for example the behavior of:
```
#!/bin/bash

for i in $* ; do
  echo '$*' $i
done

for i in "$@" ; do
  echo '$@' $i
done
```
Called with:
```
$ ./t.sh "a b" "c d"
$* a
$* b
$* c
$* d
$@ a b
$@ c d
```